### DISCUSSION TITLE
Sitemap with Hybrid AMP

### DISCUSSION THREAD

**Discussion by LucasBassetti** [vercel/next.js#16706] [Help] [answered]
Hey,

What's the right way to create the sitemap with hybrid amp? Basically I want search engines get the amp page and the users get the normal page (without amp). Is it possible? 

If yes, should add only the link with `?amp=1` in the sitemap (Ex.: `/blog/post-title?amp=1`?

Thank you!

**Comment by c0b41** [comment, marked as answer]
[Make your pages discoverable](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/discovery/)

**Reply by LucasBassetti** [reply]
Thanks @c0b41 ! It means the users will get only the normal page, right? (and the amp will be used only for SEO)
